projects/storefrontlib/src/layout/a11y/skip-link/component/skip-link.component.ts

```diff
@@ -0,0 +1,17 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { SkipLinkService } from '../service/skip-link.service';
+
+@Component({
+  selector: 'cx-skip-link',
+  templateUrl: './skip-link.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class SkipLinkComponent {
+  links$ = this.skipLinkService.skippers;
```

Comment:
as stated in the service, I like to couple with a method. And stick to `skipLinks` as the term.